codecov.yml

```
@@ -0,0 +1,8 @@
+codecov:
+  require_ci_to_pass: true
```

Let's do this to reduce the flakiness:
```suggestion
  notify:
    after_n_builds: 8  # the number of individual jobs in the test matrix
    wait_for_ci: yes
  require_ci_to_pass: true
```